src/hypermedia-constants.js

```diff
@@ -347,6 +347,13 @@ export const Classes = {
 		issueDate: 'issue-date',
 		startDate: 'start-date'
 	},
+	availabilityDates: {
+		availabilityDates: 'availability-dates',
+		dueDate: 'due-date',
+		endDate: 'end-date',
+		issueDate: 'issue-date',
+		startDate: 'availability-start-date'
+	},
```

Comment:
I don't think this maps over to the entity correctly, however, my knowledge around siren is limited.
Based on the [serializer](https://search.d2l.dev/xref/lms/le/activities/D2L.LE.Activities.Extensibility/Api/Hypermedia/ActivitiesSirenSerializers.cs?r=5bec958a#163-175) I think we only need:
- availabilityDates
- startDate
- endDate
```suggestion
	availabilityDates: {
		availabilityDates: 'availability-dates',
		endDate: 'availability-end-date',
		startDate: 'availability-start-date'
	},
```

Reply:
Ah yes, I copied the old dates one at the start because I didnt know what the schema would look like.